app/Http/Controllers/Api/AirlineController.php

```diff
@@ -7,12 +7,19 @@
 use App\Http\Requests\StoreAirlineRequest;
 use App\Http\Requests\UpdateAirlineRequest;
 use App\Models\Airline;
-use Illuminate\Http\Request;
 
 class AirlineController extends Controller
 {
     public function index(GetAirlinesRequest $request)
     {
+        $all = $request->boolean('all', false);
+
+        if ($all) {
```

Comment:
in my opinion , you are trying to make a single endpoint handle many cases. I do think that a get request should have a body either. 
The cleanest way out is by making new endpoints

Reply:
and if you want to filter, take into consideration using query params